WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/BackupDownloadActivity.kt

```diff
@@ -21,52 +34,94 @@ class BackupDownloadActivity : LocaleAwareActivity() {
         setContentView(R.layout.backup_download_activity)
 
         setSupportActionBar(toolbar_main)
-        supportActionBar?.let {
-            it.setHomeButtonEnabled(true)
-            it.setDisplayHomeAsUpEnabled(true)
-            // todo: annmarie - this is here to get past lint - this will be set from childfrag
-            it.title = getString(R.string.backup_download_details_page_title)
-        }
-        setupViewModel()
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
+
+        initViewModel(savedInstanceState)
     }
 
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
         if (item.itemId == android.R.id.home) {
             onBackPressed()
             return true
         }
-        return super.onOptionsItemSelected(item)
+        return false
+    }
+
+    override fun onBackPressed() {
+        viewModel.onBackPressed()
     }
 
-    private fun setupViewModel() {
+    override fun onSaveInstanceState(outState: Bundle) {
+        super.onSaveInstanceState(outState)
+        viewModel.writeToBundle(outState)
+    }
+
+    private fun initViewModel(savedInstanceState: Bundle?) {
         viewModel = ViewModelProvider(this, viewModelFactory)
                 .get(BackupDownloadViewModel::class.java)
-        viewModel.start()
 
-        // todo: annmarie temporary start
-        val fragment = BackupDownloadDetailsFragment.newInstance()
-        showFragment(fragment, BackupDownloadDetailsFragment.TAG, slideIn = false, isRootFragment = true)
+        viewModel.navigationTargetObservable
+                .observe(this, { target ->
```

Comment:
Minor (🔍 ): Consider moving the `.observer(...)` to the line above, just like it is done for the rest of the observables, to keep this one consistent as well.

Reply:
In 211c480